moto/elbv2/models.py

```diff
@@ -833,6 +837,11 @@ def create_target_group(self, name, **kwargs):
         self.target_groups[target_group.arn] = target_group
         return target_group
 
+    def convert_and_validate_certificates(self, certificate):
+
+        # transform default certificate  to confirm with the rest of the code and XML templates
```

Comment:
`confirm` or `conform`?
```suggestion
        # transform default certificate  to conform with the rest of the code and XML templates
```